src/en/developer-layers.md

```diff
@@ -0,0 +1,153 @@
+
+ Title: Layers for Charm Authoring
+
+# Why build from layers
+
+When creating a charm, you always have the option of doing it by
+hand writing.  That is, you can create each hook, implement the side of the
+interface you need for each relation your charm requires or provides, manage the
+dependencies, such as [charm-helpers][https://pythonhosted.org/charmhelpers/],
```

Comment:
This link didn't render for me properly.  I think you need parenthesis rather than brackets around the link.